packages/client/src/fuature/forum/components/notification/components/AlertButton/AlertButton.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { MouseEventHandler } from 'react'
+import styles from './AlertButton.module.scss';
+import classNames from 'classnames'
+type AlertButtonT = {
+  children?: React.ReactNode,
+  onClick: () => void | Promise<void>,
+  type: string;
```

Comment:
Лучше типизировать тип:
```suggestion
  type: 'success' | 'error';
```